extension/src/paymentHandler/make-payment.handler.js

```diff
@@ -40,16 +41,12 @@ async function execute(paymentObject) {
     if (action) actions.push(action)
   }
 
-  const paymentKey = paymentObject.key
-  // ensure the key is a string, otherwise the error with "code": "InvalidJsonInput" will return by commercetools API.
-  const reference = requestBodyJson.reference?.toString()
-  // ensure the key and new reference is different, otherwise the error with
-  // "code": "InvalidOperation", "message": "'key' has no changes." will return by commercetools API.
-  if (reference !== paymentKey)
-    actions.push({
-      action: 'setKey',
-      key: reference,
-    })
+  const updatePaymentAction = getPaymentKeyUpdateAction(
+    paymentObject.key,
+    request,
+    response
```

Comment:
```suggestion
    adyenRequest,
    adyenResponse
```

Reply:
why do we change name here? We need to use `request` and `response` variables. Or do you mean changing to `adyenRequest` and `adyenResponse` in the callee function arguments?

Reply:
If you check the signature of this method, there is no request coming in, just a `paymentObject`. So what kind of request and response is this?

Reply:
request and response from makePayment
https://github.com/commercetools/commercetools-adyen-integration/blob/04e374968639fa195a7d776439dec1331aa09e31/extension/src/paymentHandler/make-payment.handler.js#L18

I ever tried to modify `web-component-service.js` and rename it as below

```
const { makePaymentRequest, makePaymentResponse } = await makePayment(
    adyenMerchantAccount,
    commercetoolsProjectKey,
    makePaymentRequestObj
  )
```

 but it fails a lot of unit test cases and integration test cases. So I would rather leave it like this.